src/events/password_actions.py

```diff
@@ -1,73 +0,0 @@
-#!/usr/bin/env python3
-# Copyright 2023 Canonical Ltd.
-# See LICENSE file for licensing details.
-
-"""Event handlers for password-related Juju Actions."""
```

Comment:
why going away?

Reply:
This has never done anything else but setting a secret that the charm associated with a "fiktive" user that never existed.